internal/config/generate/main.go

```diff
@@ -27,20 +27,86 @@ func main() {
 	config := decodeTOML(data)
 	envs := sortConfig(config)
 
+	for _, env := range envs {
+		env.validate()
+	}
+	writeDoc(envs)
+	writeCode(envs)
+}
+
+func writeCode(envs []Env) {
 	var code strings.Builder
-	var doc strings.Builder
 
 	addCodeHeader(&code)
-	addDocHeader(&doc)
 
-	addLine(&doc, "<!-- markdownlint-disable MD012 -->")
+	//Validate envs
+	for i := range envs {
```

Comment:
`for _, env := range envs {`

Reply:
as Validate sets the default value, if we use `_,env` we get a copy of the "Env" and the value is not correctly set. Unless we do `.validate()` on every loop and use this Env instance to generate that code snippet.